test/DotNetty.Transport.Tests.Performance/Utilities/CounterHandlerInbound.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace DotNetty.Transport.Tests.Performance.Utilities
+{
+    using DotNetty.Transport.Channels;
+    using NBench;
+
+    class CounterHandlerInbound : ChannelHandlerAdapter
+    {
+        readonly Counter throughput;
```

Comment:
is it thread-safe? If not, then it is not a good idea to make this handler shareable.